Assets/MXR.SDK/Runtime/IMXRSystem.cs

```diff
@@ -148,6 +148,18 @@ public interface IMXRSystem {
         /// <param name="password">The password to use to attempt to connect</param>
         void ConnectToWifiNetwork(string ssid, string password);
 
+        ///// <summary>
+        ///// Connects to an Enterpirse wifi network
+        ///// </summary>
+        ///// <param name="ssid">The SSID of the network to connect to</param>
+        ///// <param name="password">The password to use to attempt to connect</param>
+        ///// <param name="identity">The identity to use to attempt to connect</param>
+        ///// <param name="eapMethod">The EAP Method to use to attempt to connect</param>
+        ///// <param name="phase2AuthenticationMethod">The phase 2 Authentication Method to use to attempt to connect</param>
+        ///// <param name="anonymousIdentity">The Anonymous Identity to use to attempt to connect</param>
+        ///// <param name="domain">The domain to use to attempt to connect</param>
+        void ConnectToWifiEnterpriseNetwork(string ssid, string password, string identity, string eapMethod, string phase2AuthenticationMethod, string anonymousIdentity, string domain);
```

Comment:
When you have this many parameters, it is often useful to encapsulate them in an object. If everything is the same type, it is really easy to accidentally put password where identity goes and vice versa.

Here's an articular on parameter objects: https://java-design-patterns.com/patterns/parameter-object/

Also - some of these parameters are actually Enums. Making enum types with explanations would make this interface cleaner.  We have enums on the backend / frontend, with explanations in the Wifi field here. You may also find the comments about if one field is present what ie means for others useful: https://github.com/ManageXR/mighty-api/blob/2b6f365ff5f95e055ff68c7bf7dc66b3fd052a27/packages/common/src/data/types/WifiNetwork.ts#L4 

Remember also that the SDK is public, and customer facing. so - EAP method as a string could be 'peap' or 'PEAP' and there is no way for clients to know without an enum to guide them (or stricter documentation).

Reply:
Yes I agree, this is only temporary. I have [this](https://github.com/ManageXR/custom-launcher/blob/e0b78fd606c53360a11406acb611822a71820476/Assets/_Application/Scripts/EnterpriseWifiConnectionRequest.cs) class for the request.

Ok got it, I'll change them to enums